addon/ng2/blueprints/component-test/index.js

```diff
@@ -1,12 +1,12 @@
 module.exports = {
   description: '',
-  
+
   // ******************************************************
   // ******************************************************
   // LEAVE THIS HERE
   // Must override install to prevent ember's component tests
   // ******************************************************
   // ******************************************************
-  
-  install: function(options){}
+
+  install: function(options) {}// eslint-disable-line no-unused-vars
```

Comment:
if you name the argument `_` would it still complain?

Or remove the argument.